docs/users/author-guide.md

```diff
@@ -1,28 +1,106 @@
+
+- Models (e.g., in CellML, SBML format)
+- Simulation experiments (SED-ML files)s
+- Visualizations for visualizing simulation results (e.g., in Vega format)
+- Supplementary files, such as data used to calibrate the model
+- Metadata about the simulation project (RDF files that follow the OMEX Metadata guidelines)
+ 
 ## Validating projects
 
+BioSimulations provides a validation tool for validating the contents of a COMBINE/OMEX archive. This tool is available on [runBioSimulations](https://run.biosimulations.org/utils/validate-project). The user can upload a COMBINE/OMEX archive or provide a URL to a COMBINE/OMEX archive. The tool will validate the contents of the archive and report any errors. The interface provides a customizable report of the validation results, including validation of the model file, Sed-ML file, metadata, archive manifest, and any images presented in the archive.
+
+![validation-tool](../images/validate.png)
+
+!!!warning
+    This application can only validate targets for simulation observables for unchanged models. Targets for modified models cannot be validated statically, independently from executing the associated SED-ML document. Simulation tools validate such targets when they execute their parent SED-ML document. For the same reason, this application cannot validate targets for model changes. Simulation tools validate such model change targets during their execution.
+
+
+
+This validation can also be accessed programmatically from the [BioSimulations API](https://combine.api.biosimulations.org).
+
+
+
+
 ## Using BioSimulators to test projects prior to submission to BioSimulations
 
+Authors can run their simulation projects locally using the BioSimulators docker images. To run the project locally, pull the appropriate BioSimulators docker image, and then run the simulator as follows:
+
+```bash
+docker run ghcr.io/biosimulators/tellurium:2.2.1 -v /path/to/project:/root -v /path/to/output:/root/out -i project.omex -o /root/out
+```
+!!!note
+    Ensure that the directories containing the project.omex file and the output directory are mounted in the container. For more information, see the Docker documentation [here](https://docs.docker.com/storage/bind-mounts/).
+
 ## Executing projects
 
+Before submitting a project, the simulation must be executed using runBioSimulations. This tool provides a web interface for executing a simulation project. The user can upload a COMBINE/OMEX archive or provide a URL to a COMBINE/OMEX archive. The tool will execute the simulation and report any errors. The interface provides a customizable view of the simulation results and simulation logs. 
+
+
 ## Reviewing simulation results before publication
 
+Before publishing a simulation project, the user should review the simulation results. The author should verify the following: 
+
+1. The simulation runs successfully and all outputs are present.
```

Comment:
I think this order would be easier to follow:

- Project includes all intended files
- Project executed successfully
- Outputs are as intended
- Metadata

Reply:
Agreedm thanks